platform-db/src/main/java/com/softicar/platform/db/runtime/table/logic/AbstractDbTableRowFieldAccessor.java

```diff
@@ -29,7 +30,7 @@ private void reloadRowIfNecessary() {
 
 		if (!row.table().getPrimaryKey().containsField(field) && !row.impermanent()) {
 			if (row.stub() || row.invalidated()) {
-				row.reload();
+				row.table().refreshAll(Collections.singleton(row));
```

Comment:
Hm, is this a common occurence that we refresh a single row?
If it is, maybe we should add a default implementation of `refreshAll` with a varArgs-Param

Reply:
As this is rather uncommon (beyond unit tests), I don't really see a point in adding yet another method to `IDbBasicTable`.